easybuild/tools/module_generator.py

```diff
@@ -133,6 +132,15 @@ class ModuleGenerator:
     # a single level of indentation
     INDENTATION = ' ' * 4
 
+    # shell environment variable name: ${__}VAR_NAME_00_SUFFIX
+    REGEX_SHELL_VAR_PATTERN = r'[A-Z_]+[A-Z0-9_]+'
```

Comment:
Just asking: strictly speaking environment variables don't have to be all caps, that's just a convention.

Are we happy with not recognizing lowercase environment variables like `$THIS_is_an_example`?

Reply:
For now, we'll only supports "proper" environment variables names, i.e. in all caps.